### PR TITLE
add intro to readme

### PR DESCRIPTION
**Pull request by onlybaby**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# GitDummy
\ No newline at end of file
+# GitDummy
```
Comment (BozhaoJin) [inline README.md:1]: Please fix this.

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
+This is our Pretty Office!
```
Comment (BozhaoJin) [inline README.md]: nitpick. Please add an extra line after the heading.

**Review by BozhaoJin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (onlybaby) [reply README.md:1]: it is not my problem!

Comment (BozhaoJin) [reply README.md]: Due to inappropriate attitude and behavior, this internship has been concluded as a total failure and a complete fiasca. Your return offer has now been revoked. Please leave the office asap.

**Review by BozhaoJin — APPROVED** [review]

**Pull request merged**